### PR TITLE
[post][Go: tasks repeated at intervals]

### PR DESCRIPTION
**Pull request by codeworm96**
## Post info

- Title: Go: tasks repeated at intervals
- Author: codeworm96
- License: default
- Estimate finish time: finished

### CODE REVIEW

**Comment by at15** [comment]
excited

File: source/go:_tasks_repeated_at_intervals.md [hunk 1]
```diff
@@ -0,0 +1,123 @@
+title: "Go: tasks repeated at intervals"
+date: 2017-10-26 05:35:01 +0000
+update: 2017-10-26 05:35:01 +0000
+author: codeworm96
+# NOTE: the image locate in `source` folder
+# cover: "-/images/posts/example.png"
+tags:
+    - golang
```
Comment (at15) [inline source/go:_tasks_repeated_at_intervals.md]: 这里把 golang 改成 Go 更好一点，之前的 golang 是我的锅，最好把我的那个 (Ayi) 也顺手改了.....

**Review by at15 — CHANGES_REQUESTED** [review]

**Comment by gaocegege** [comment]
Update
Reference

可以删掉

**Comment by gaocegege** [comment]
LGTM

Could I merge it?

[Approved with PullApprove](https://pullapprove.com/dyweb/blog/pull-request/25/?utm_source=github-pr&utm_medium=comment-badge&utm_campaign=dyweb/blog)

**Comment by at15** [comment]
@gaocegege go ahead

LGTM

[Approved with PullApprove](https://pullapprove.com/dyweb/blog/pull-request/25/?utm_source=github-pr&utm_medium=comment-badge&utm_campaign=dyweb/blog)

**Pull request merged**